docker-app/qfieldcloud/core/admin.py

```diff
@@ -71,6 +77,133 @@ def admin_urlname_by_obj(value, arg):
 admin.site.unregister(SocialAccount)
 admin.site.unregister(SocialApp)
 admin.site.unregister(SocialToken)
+admin.site.unregister(EmailAddress)
+
+UserEmailDetails = namedtuple(
+    "UserEmailDetails",
+    [
+        "id",
+        "username",
+        "first_name",
+        "last_name",
+        "type",
+        "email",
+        "date_joined",
+        "owner_id",
+        "owner_username",
+        "owner_email",
+        "owner_first_name",
+        "owner_last_name",
+        "owner_date_joined",
+    ],
+)
+
+
+class EmailAddressAdmin(EmailAddressAdmin):
+    def get_urls(self):
+        urls = super().get_urls()
+        return [
+            *urls,
+            path(
+                "admin/export_emails_to_csv",
```

Comment:
```suggestion
                "admin/export_emails_to_csv/",
```

Can we prefer the trailing slash convention, it is quite popular within Django.